config/configgrpc/configgrpc.go

```diff
@@ -46,6 +46,11 @@ type KeepaliveClientConfig struct {
 	PermitWithoutStream bool          `mapstructure:"permit_without_stream"`
 }
 
+// NewDefaultKeepaliveClientConfig() creates a new KeepaliveClientConfig with any default values set
+func NewDefaultKeepaliveClientConfig() *KeepaliveClientConfig {
+	return &KeepaliveClientConfig{}
```

Comment:
Based on https://pkg.go.dev/google.golang.org/grpc@v1.62.1/keepalive#ClientParameters I believe a default value of `10s` should be set.

Reply:
Done